lib/services.rb

```diff
@@ -12,6 +13,7 @@ def self.content_store
   end
 
   def self.email_sender
-    @email_sender ||= EmailSender::Notify.new
+    return @email_sender ||= EmailSender::Notify.new if EmailAlertAPI.config.email_service_provider == "NOTIFY"
```

Comment:
I think it would be good if could raise an exception here if something is not what we expect. We could still accept `nil` as meaning the same as `pseudo`, but I think it would still be good if an exception is raised for something else.

Reply:
added an exception to this now @thomasleese - see what you think